core/lexicon/en/file.inc.php

```diff
@@ -76,6 +76,9 @@
 $_lang['file_name'] = 'File Name';
 $_lang['file_quick_create'] = 'Quick Create File';
 $_lang['file_quick_update'] = 'Quick Edit File';
+$_lang['file_package_err_invalid_directory'] = 'Packages directory doesnt appear to exist!';
```

Comment:
```suggestion
$_lang['file_package_err_invalid_directory'] = 'Packages directory does not exist!';
```

Reply:
It should have an article—"The packages directory does not exist!"